spec/menu_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'menu'
+
+describe Menu do
+
+  let (:menu) { Menu.new }
+
+    it 'Menu should have a list of dishes' do
+      dishes = {  :Burger => 5.00,
+                  :Fries => 2.00,
+                  :Pizza => 8.00,
+                  :Wings => 3.00
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)